it/api/endpoints/bfLoss/list/v3/ListBFLossesControllerISpec.scala

```diff
@@ -103,57 +49,15 @@ class ListBFLossesControllerISpec extends V3IntegrationBaseSpec {
         response.header("Content-Type") shouldBe Some("application/json")
       }
 
-      "query without a tax year" in new Test {
```

Comment:
Has this file been the victim of a bad merge? Looks to be reverting all the changes made for the V3 changes to make multiple downstream requests.